docs/config/production/config.toml

```
@@ -0,0 +1,14 @@
+baseURL = "https://docs.k8ssandra.io/"
+url_latest_version = "https://docs.k8ssandra.io"
+
+[[params.versions]]
+  version = "k8ssandra-operator (current)"
+  fullversion = "k8ssandra-operator"
+  githubbranch = "docs"
+  url = "https://docs-temp.k8ssandra.io"
```

```suggestion
  url = "https://docs.k8ssandra.io"
```

Where did you get `docs-temp` from?

---

it was in production-v2. this is what i was going to ask it doesn't seem relevant anymore?